components/ntp_background_images/browser/features.h

```diff
@@ -7,11 +7,25 @@
 #define BRAVE_COMPONENTS_NTP_BACKGROUND_IMAGES_BROWSER_FEATURES_H_
 
 #include "base/feature_list.h"
+#include "base/metrics/field_trial_params.h"
 
 namespace ntp_background_images {
 namespace features {
+
 BASE_DECLARE_FEATURE(kBraveNTPBrandedWallpaperDemo);
+
 BASE_DECLARE_FEATURE(kBraveNTPSuperReferralWallpaper);
+
+BASE_DECLARE_FEATURE(kBraveNTPBrandedWallpaper);
+
+// Show initial branded wallpaper after nth new tab page for fresh installs.
+constexpr base::FeatureParam<int> kInitialCountToBrandedWallpaper{
+    &kBraveNTPBrandedWallpaper, "initial_count_to_branded_wallpaper", 1};
```

Comment:
As this initial value is set at the first run, I think this value doesn't need to be feature params?

Reply:
This was so that for fresh installs we have the option of changing the value

Reply:
Ah, ok for that purpose :)